src/applications/mhv/secure-messaging/components/MessageDetailBlock.jsx

```diff
@@ -75,7 +76,7 @@ const MessageDetailBlock = props => {
         id={messageId}
         threadId={threadId}
         onReply={handleReplyButton}
-        hideReplyButton={hideReplyButton}
+        hideReplyButton={cannotReply}
```

Comment:
If you want to rename this prop to match 'cannotReply' you could; your preference